### ISSUE TITLE
NodeJS Linux ARM64 tarball not found when installing from node:16-alpine Docker image

### ISSUE THREAD

**Issue by hbd** [duckdb/duckdb#6091]
### What happens?

When installing duckdb `0.6.1` in a `node:16-alpine` Docker container, I get the following error:
```
 > [5/8] RUN npm ci:
#9 20.26 npm notice
#9 20.26 npm notice New minor version of npm available! 9.3.1 -> 9.4.1
#9 20.26 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.4.1>
#9 20.26 npm notice Run `npm install -g npm@9.4.1` to update!
#9 20.26 npm notice
#9 20.26 npm ERR! code 1
#9 20.26 npm ERR! path /usr/src/app/node_modules/duckdb
#9 20.26 npm ERR! command failed
#9 20.26 npm ERR! command sh -c node-pre-gyp install --fallback-to-build
#9 20.26 npm ERR! Failed to execute '/usr/local/bin/node /usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/usr/src/app/node_module
s/duckdb/lib/binding/duckdb.node --module_name=duckdb --module_path=/usr/src/app/node_modules/duckdb/lib/binding --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_labe
l=node-v108' (1)
#9 20.26 npm ERR! node-pre-gyp info it worked if it ends with ok
#9 20.26 npm ERR! node-pre-gyp info using node-pre-gyp@1.0.10
#9 20.26 npm ERR! node-pre-gyp info using node@18.14.0 | linux | arm64
#9 20.26 npm ERR! node-pre-gyp info check checked for "/usr/src/app/node_modules/duckdb/lib/binding/duckdb.node" (not found)
#9 20.26 npm ERR! node-pre-gyp http GET https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
#9 20.26 npm ERR! node-pre-gyp ERR! install response status 403 Forbidden on https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
#9 20.26 npm ERR! node-pre-gyp WARN Pre-built binaries not installable for duckdb@0.6.1 and node@18.14.0 (node-v108 ABI, musl) (falling back to source compile with node-gyp)
#9 20.26 npm ERR! node-pre-gyp WARN Hit error response status 403 Forbidden on https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
#9 20.26 npm ERR! gyp info it worked if it ends with ok
#9 20.26 npm ERR! gyp info using node-gyp@9.3.0
#9 20.26 npm ERR! gyp info using node@18.14.0 | linux | arm64
#9 20.26 npm ERR! gyp info ok
#9 20.26 npm ERR! gyp info it worked if it ends with ok
#9 20.26 npm ERR! gyp info using node-gyp@9.3.0
#9 20.26 npm ERR! gyp info using node@18.14.0 | linux | arm64
#9 20.26 npm ERR! gyp ERR! find Python
#9 20.26 npm ERR! gyp ERR! find Python Python is not set from command line or npm configuration
#9 20.26 npm ERR! gyp ERR! find Python Python is not set from environment variable PYTHON
#9 20.26 npm ERR! gyp ERR! find Python checking if "python3" can be used
#9 20.26 npm ERR! gyp ERR! find Python - "python3" is not in PATH or produced an error
#9 20.26 npm ERR! gyp ERR! find Python checking if "python" can be used
#9 20.26 npm ERR! gyp ERR! find Python - "python" is not in PATH or produced an error
#9 20.26 npm ERR! gyp ERR! find Python
#9 20.26 npm ERR! gyp ERR! find Python **********************************************************
#9 20.26 npm ERR! gyp ERR! find Python You need to install the latest version of Python.
#9 20.26 npm ERR! gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
#9 20.26 npm ERR! gyp ERR! find Python you can try one of the following options:
#9 20.26 npm ERR! gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
#9 20.26 npm ERR! gyp ERR! find Python   (accepted by both node-gyp and npm)
#9 20.26 npm ERR! gyp ERR! find Python - Set the environment variable PYTHON
#9 20.26 npm ERR! gyp ERR! find Python - Set the npm configuration variable python:
#9 20.26 npm ERR! gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
#9 20.26 npm ERR! gyp ERR! find Python For more information consult the documentation at:
#9 20.26 npm ERR! gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
#9 20.26 npm ERR! gyp ERR! find Python **********************************************************
#9 20.26 npm ERR! gyp ERR! find Python
... etc
```

Should there be a tarball at `https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz`?
I also get this when using the `node:18-alpine` image.

### To Reproduce

1. In Docker container running the `node:16-alpine` image
2. Run `npm install duckdb`

### OS:

Mac M1 (arm64)

### DuckDB Version:

0.6.1

### DuckDB Client:

NodeJS

### Full Name:

Zak Keener

### Affiliation:

Tydo

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by coreyfournier** [comment]
The error says you need python. Did you install python in your build image?

**Comment by tnightengale** [comment]
Bump. @coreyfournier The main issue is not that python is missing. The issue is that the binary does not exist. These are the critical lines:
```
#9 20.26 npm ERR! node-pre-gyp http GET https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
#9 20.26 npm ERR! node-pre-gyp ERR! install response status 403 Forbidden on https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
```

@Mause I saw this issue also arise in #5159. Do you know why these binaries are not being built?

**Comment by Mause** [comment]
> Bump. @coreyfournier The main issue is not that python is missing. The issue is that the binary does not exist. These are the critical lines:
> ```
> #9 20.26 npm ERR! node-pre-gyp http GET https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
> #9 20.26 npm ERR! node-pre-gyp ERR! install response status 403 Forbidden on https://duckdb-node.s3.amazonaws.com/duckdb-v0.6.1-node-v108-linux-arm64.tar.gz
> ```
> 
> @Mause I saw this issue also arise in #5159. Do you know why these binaries are not being built?

There was an issue with the 0.7.0 build, Mac m1 prebuilds should now be available for 0.7.1
